model/gen-ai/metrics.yaml

```diff
@@ -37,6 +37,8 @@ groups:
     attributes:
       - ref: gen_ai.openai.response.service_tier
         requirement_level: recommended
+      - ref: gen_ai.openai.response.system_fingerprint
```

Comment:
we have a bug that prevents MD table generation - https://github.com/open-telemetry/semantic-conventions/pull/1535, let me get than one merged and then you'll need to re-generate tables again. Sorry for the inconvenience